caliban/platform/cloud/core.py

```diff
@@ -540,22 +558,25 @@ def submit_job_specs(
   execute_requests(requests, num_specs, num_retries=request_retries)
 
 
-def submit_ml_job(job_mode: conf.JobMode,
-                  docker_args: Dict[str, Any],
-                  region: ct.Region,
-                  project_id: str,
-                  credentials_path: Optional[str] = None,
-                  dry_run: bool = False,
-                  job_name: Optional[str] = None,
-                  machine_type: Optional[ct.MachineType] = None,
-                  gpu_spec: Optional[ct.GPUSpec] = None,
-                  tpu_spec: Optional[ct.TPUSpec] = None,
-                  image_tag: Optional[str] = None,
-                  labels: Optional[Dict[str, str]] = None,
-                  experiment_config: Optional[ce.ExpConf] = None,
-                  script_args: Optional[List[str]] = None,
-                  request_retries: Optional[int] = None,
-                  xgroup: Optional[str] = None) -> None:
+def submit_ml_job(
+    job_mode: conf.JobMode,
+    docker_args: Dict[str, Any],
+    region: ct.Region,
+    project_id: str,
+    credentials_path: Optional[str] = None,
+    dry_run: bool = False,
+    job_name: Optional[str] = None,
+    machine_type: Optional[ct.MachineType] = None,
+    gpu_spec: Optional[ct.GPUSpec] = None,
+    tpu_spec: Optional[ct.TPUSpec] = None,
+    image_tag: Optional[str] = None,
+    labels: Optional[Dict[str, str]] = None,
+    experiment_config: Optional[ce.ExpConf] = None,
+    script_args: Optional[List[str]] = None,
+    request_retries: Optional[int] = None,
+    xgroup: Optional[str] = None,
+    env: Optional[Dict[str, str]] = {},
```

Comment:
don't use a map directly, please follow the convention for other args and use `None`

Reply:
ok